src/librustdoc/html/static/css/rustdoc.css

```diff
@@ -2056,9 +2056,19 @@ in storage.js plus the media query with (min-width: 701px)
 }
 
 @media print {
-	nav.sub, .content .out-of-band {
+	nav.sidebar, nav.sub, .content .out-of-band, a.srclink, #copy-path,
+	details.rustdoc-toggle[open] > summary::before, details.rustdoc-toggle > summary::before,
+	details.rustdoc-toggle.top-doc > summary {
 		display: none;
 	}
+
+	.docblock {
+		margin-left: 0;
+	}
+
+	main {
+		padding-left: 15px;
```

Comment:
Don't you want padding on both sides?

Reply:
default style already contains padding of equal size on the other side

Reply:
good point, i'll just set them all explicitly